onnxtech.txt

```diff
@@ -3,165 +3,394 @@ apis: c
 architectures: gpu: nvidia
-architectures: ibm power
-architectures: ibm power: power10
-architectures: npu
-architectures: vad
-architectures: vpu
-architectures: x64
-architectures: x86
-architectures: x86: amx
-azure
+architectures: gpu: rocm
 execution providers
+execution providers: acl
 execution providers: armnn
+execution providers: azure
+execution providers: cann
 execution providers: coreml
+execution providers: cpu
 execution providers: cuda
-execution providers: cuda: nccl
-execution providers: direct ml
-execution providers: ep
+execution providers: directml
+execution providers: dnnl
+execution providers: js
 execution providers: migraphx
 execution providers: nnapi
-execution providers: onednn
-execution providers: openvino: ov
-execution providers: provider
+execution providers: openvino
+execution providers: qnn
+execution providers: rknpu
 execution providers: rocm
-execution providers: rocm: rocblas
+execution providers: shared
+execution providers: snpe
 execution providers: tensorrt
 execution providers: tvm
-featurizers
+execution providers: vitisai
+execution providers: webnn
+execution providers: winml
+execution providers: xnnpack
 graph
-graph: graph processing: 3d unet
-graph: graph processing: abseil
-graph: graph processing: edge
-graph: graph processing: gist
-graph: graph processing: weight
+graph: graph processing
 graph: optimization
 graph: partition
-graph: protobuf
+graph: serialization
 inferencing
 models
-models: customer model
-models: learning models
+models: custom model
+models: learning model
 models: model conversion
-models: model graphs
 models: model processing
 models: model processing: fusion
 models: model support
 models: model testing
 models: model transformation
 models: model types
-models: model types: beamsearch
-models: model types: bert
-models: model types: bfloat16
+models: model types: transformer-based: bert
 models: model types: convolution
-models: model types: convolution: cnn
-models: model types: deepthink longformer
-models: model types: float16
+models: model types: transformer-based: deepthink longformer
 models: model types: mtdnn
-models: model types: nuphar
+execution providers: tvm: nuphar
 models: model types: rcnn
 models: model types: resnet
 models: model types: stable diffusion
 models: model types: xnnpack
-models: model types:bfloat16
 models: model zoo
-models: onnx models
-models: ort model
 models: testing models
 models: training models
 models: training models: fairscale
 models: training models: gathergrad
-models: training models: grad
-models: training models: gradient
-models: training models: pytorch
-models: training models: tuning
-models: transformer-based
-nodes/operators/kernels/tensors
-nodes/operators/kernels/tensors :clip
-nodes/operators/kernels/tensors: aten
-nodes/operators/kernels/tensors: attention
-nodes/operators/kernels/tensors: bias
-nodes/operators/kernels/tensors: convgrad
-nodes/operators/kernels/tensors: convtranspose
```

Comment:
It's not entirely clear to me if we need each op type as a separate concept, but if we do, we should probably have them all, for completeness, I've split them out into their own top-level